draft-ietf-privacypass-protocol.md

```diff
@@ -579,7 +576,7 @@ consistency mechanism to help ensure they are not being given unique keys; see
 
 ## Token Type
 
-This document updates the "Token Type" Registry with the following values.
+This document updates the "Token Type" Registry from {{AUTHSCHEME, Section 5.2}} with the following values.
```

Comment:
You're also missing the `Nid: The length of the token key identifier` that the auth scheme says is required to be populated

Reply:
Good catch!

Reply:
Done.